21_node_management/22_search_nodes.txt

```diff
@@ -10,25 +10,154 @@ about the Nodes which have been already validated, and are managed by Rudder.
 
 ==== Quick Search
 
-The easiest search tool is the Quick search: type in the search field the first
-letters of the 'Rudder ID', 'Reference', or 'Hostname'; choose the accurate Node in
-the autocompletion list; validate and look at the Node information. This search
-tool can be very useful to help you create a new search in the Advanced Search.
-
-.Quick search the Node called +debian-node+
+You might have noticed the small text area at the top of the Rudder interface:
+it is the Quick Search bar. Its purpose is to enable a user to easily search for
+Rudder elements (Nodes, Groups, Directives, Parameters, Rules)  based on their
+name, id, description, inventory...
+
+image::./images/rudder-quicksearch.png[Quicksearch]
+
+An autocompletion list will appear as soon as Rudder detects an element it can
+identify, you just have to click on it to be redirected to the element
+configuration page.
+
+More complex search queries can be input using the "in:" and "is:" keywords,
+"is" targeting a Rudder category and "in" targeting elements like name,
```

Comment:
Careful with the use of the term "category" here. Categories in Rudder are used to sort groups and rules. Let's replace this with "is targets Rudder objects by type, and in..."